presto-kafka/src/test/java/com/facebook/presto/kafka/util/EmbeddedKafka.java

```diff
@@ -131,28 +138,16 @@ public void createTopics(int partitions, int replication, Properties topicProper
         }
     }
 
-    public CloseableProducer<Long, Object> createProducer()
+    public KafkaProducer<Long, Object> createProducer()
     {
-        Map<String, String> properties = ImmutableMap.<String, String>builder()
-                .put("metadata.broker.list", getConnectString())
-                .put("serializer.class", JsonEncoder.class.getName())
-                .put("key.serializer.class", NumberEncoder.class.getName())
-                .put("partitioner.class", NumberPartitioner.class.getName())
-                .put("request.required.acks", "1")
-                .build();
-
-        ProducerConfig producerConfig = new ProducerConfig(toProperties(properties));
-        return new CloseableProducer<>(producerConfig);
-    }
-
-    public static class CloseableProducer<K, V>
-            extends Producer<K, V>
-            implements AutoCloseable
-    {
-        public CloseableProducer(ProducerConfig config)
-        {
-            super(config);
-        }
+        Properties properties = new Properties();
+        properties.setProperty(BOOTSTRAP_SERVERS_CONFIG, getConnectString());
+        properties.setProperty(VALUE_SERIALIZER_CLASS_CONFIG, JsonEncoder.class.getName());
+        properties.setProperty(KEY_SERIALIZER_CLASS_CONFIG, LongSerializer.class.getName());
+        properties.setProperty(PARTITIONER_CLASS_CONFIG, DefaultPartitioner.class.getName());
```

Comment:
Default partitioner is default. Need to specify?

Reply:
Fixed. Thanks for catching this one.